inference-engine/thirdparty/movidius/mvnc/src/mvnc_api.c

```diff
@@ -3151,25 +3151,22 @@ ncStatus_t ncFifoDestroy(struct ncFifoHandle_t ** fifoHandle)
     CHECK_MUTEX_SUCCESS(pthread_mutex_lock(&d->graph_stream_m));
     rc = trySendCommand(d->graph_monitor_stream_id, &cmd, sizeof(cmd));
     if (rc != 0) {
-        CHECK_MUTEX_SUCCESS(pthread_mutex_unlock(&d->graph_stream_m));
         mvLog(MVLOG_WARN, "can't send command\n");
         rc = NC_ERROR;
     } else if (checkGraphMonitorResponse(d->graph_monitor_stream_id)) {
-        CHECK_MUTEX_SUCCESS(pthread_mutex_unlock(&d->graph_stream_m));
         mvLog(MVLOG_WARN, "myriad NACK\n");
         rc = NC_ERROR;
     }
     CHECK_MUTEX_SUCCESS(pthread_mutex_unlock(&d->graph_stream_m));
 
     CHECK_MUTEX_SUCCESS(pthread_mutex_lock(&d->dev_data_m));
     if (deallocateFifo(handle)) {
-        CHECK_MUTEX_SUCCESS(pthread_mutex_unlock(&d->dev_data_m));
```

Comment:
I think the warning will not be superfluous:)

Reply:
Added